app/metrics.yaml

```diff
@@ -223,6 +224,25 @@ events:
     notification_emails:
       - fenix-core@mozilla.com
     expires: "2021-07-01"
+  set_default_browser_experiment:
+    type: event
+    description: |
+      Indicates which branch let to user to set Firefox as a default browser
```

Comment:
@tdsmith this could covered #18857.

Reply:
Thanks. I think we also need something that isn't specific to these treatment branches; I left a comment there.

Reply:
👍🏽

Reply:
Do we want an event to identify which was the experiment that resulted on users changing their default browser?

Reply:
I would like to know if we want to keep this metric?

Reply:
I think it's not necessary. Sorry, I should have responded to Elise's comment. 

We have the branch information in the ping and the key outcome measure is set-default by any mechanism, so it's not necessary for the analysis. This could help us diagnose any surprises but I think we can replicate the heuristic here using data in telemetry.

Reply:
Thanks for the clarification :)